cmd/notation/plugin.go

```diff
@@ -62,3 +108,169 @@ func listPlugins(command *cobra.Command) error {
 	}
 	return tw.Flush()
 }
+
+func installPlugin(command *cobra.Command, args []string, force bool) error {
+	if len(args) != 1 {
+		return errors.New("missing plugin package")
+	}
+
+	pluginPath := args[0]
+
+	var pluginBinary string
+	// if plugin contains a file path split and select the last element in the array
+	if strings.Contains(pluginPath, "/") {
+		pluginBinary = strings.Split(pluginPath, "/")[len(strings.Split(pluginPath, "/"))-1]
+	} else {
+		pluginBinary = pluginPath
+	}
+
+	// get plugin metadata
+	cmd := exec.Command("./"+pluginPath, "get-plugin-metadata")
+
+	output, err := cmd.Output()
+	if err != nil {
+		panic(err)
+	}
+
+	var newPlugin map[string]interface{}
+	err = json.Unmarshal([]byte(output), &newPlugin)
+	if err != nil {
+		return err
+	}
+
+	pluginName := newPlugin["name"].(string)
+	newPluginVersion := newPlugin["version"].(string)
+	newSemVersion, err := semver.NewVersion(newPluginVersion)
+	if err != nil {
+		return err
+	}
+
+	// get plugin directory
+	pluginDir, err := dir.PluginFS().SysPath(pluginName)
+	if err != nil {
+		panic(err)
+	}
+
+	// Check if plugin directory exists
+	_, err = os.Stat(pluginDir)
+
+	// create the directory, if not exist
+	if os.IsNotExist(err) {
+		if err := os.MkdirAll(pluginDir, 0755); err != nil {
+			return err
+		}
+	}
+
+	// Check if plugin exists
+	_, err = os.Stat(pluginDir + "/" + pluginBinary)
+
+	// copy plugin, if not exist
+	if os.IsNotExist(err) {
+		copyPlugin(pluginPath, pluginDir+"/"+pluginBinary)
+	}
+
+	// overwrite plugin, if force flag is set
+	if err == nil && force {
```

Comment:
if `err != nil` is handled above, then it's just `if force {xxx}` here.

Reply:
`if force` logic has been [replaced](https://github.com/notaryproject/notation/pull/617/commits/3eb146c202f2e6a75d4184f5c79d43cd42bc80e5#diff-2ce9482b6436893530d00c1cf4d9cd0231854ce00b30bc668ee63b16fca4b373R152).